ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -7882,4 +7882,16 @@ DTPM:
   - detergent_pump_run_status
   - detergent_pump_run_command
   implements:
-  - MONITORING
\ No newline at end of file
+  - MONITORING
+
+ZDHC:
+  guid: "d50c12f3-98bf-4da2-bc36-57d44160c789"
+  description: "zone dewpoint control with chilled water cooling and electric reheat"
+  is_abstract: true
+  uses:
+  - zone_air_dewpoint_temperature_sensor
+  - zone_air_dewpoint_temperature_setpoint
+  - chilled_water_valve_percentage_command
+  - heating_percentage_command
```

Comment:
I would separate the heating control from here - you can probably use the existing HTSC type

Reply:
No